src/quo2/components/wallet/account_origin/component_spec.cljs

```diff
@@ -0,0 +1,100 @@
+(ns quo2.components.wallet.account-origin.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.core :as quo]
+            [utils.i18n :as i18n]))
+
+(h/describe
+  "account origin tests"
```

Comment:
[Code style] The `describe` text arg almost always go on the same line as `describe`.

Reply:
Done